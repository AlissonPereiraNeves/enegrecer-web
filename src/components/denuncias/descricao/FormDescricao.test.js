import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Descricao from './FormDescricao';

describe('Form Descrição', () => {
  it('deve seguir a estrutura definida ', () => {
    const desricao = shallow(<Descricao />);
    const desricaoJson = toJson(desricao);
    expect(desricaoJson).toMatchSnapshot();
  });
});
