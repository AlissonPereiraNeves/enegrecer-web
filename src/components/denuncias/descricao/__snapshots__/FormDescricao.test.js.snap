// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Descrição deve seguir a estrutura definida  1`] = `
<div>
  <br />
  <p>
    Descrição da agressão
  </p>
  <br />
  <h4>
     Compreendemos o quanto é difícil externalizar a agressão em palavras.
  </h4>
  <p>
    Mas, é importante para que futuramente consigamos trabalhar para que essas agressões tenham um fim. 
  </p>
  <div
    className="row"
  >
    <combobox
      divClasse="col s6"
      id="periodo"
      itens={
        Array [
          " ",
          "Manhã",
          "Tarde",
          "Noite",
        ]
      }
      label="Período*"
      state="periodoAgressao"
      valorPadrao=""
    />
    <Data
      divClasse="input-field col s6"
      id="data-agressao"
      label="Data de Agressão"
      state="dataAgressao"
    />
  </div>
  <div
    className="row"
  >
    <comboEstado
      classes=" col s6"
      id="descricao-estado"
      state="estadoAgressor"
    />
    <campoTexto
      divClasse="input-field col s6"
      id="cidade-descricao"
      label="Cidade*"
      maxLen={40}
      placeholder=""
      state="cidadeAgressao"
      type="text"
    />
  </div>
  <div
    className="row"
  >
    <campoTexto
      divClasse="input-field col s6"
      id="bairro-descricao"
      label="Bairro*"
      maxLen={40}
      placeholder=""
      state="bairroAgressor"
      type="text"
    />
    <campoTexto
      divClasse="input-field col s6"
      id="endereco-descricao"
      label="Endereço"
      maxLen={40}
      placeholder=""
      state="enderecoAgressor"
      type="text"
    />
  </div>
  <div
    className="row"
  >
    <campoTexto
      divClasse="input-field col s12"
      id="descricao-denuncia"
      inputClasse="materialize-textarea"
      label="Descrição da denúncia"
      maxLen={255}
      state="descricaoDenuncia"
      type="text"
    />
  </div>
</div>
`;
